data/info/009_threads/embeds/01/description.md

```diff
@@ -14,9 +14,10 @@
 <#940904445343060028> (Fandom memes) - memes connected with fandom and its members but not directly with the Adventures of Reksio series.
 <#924590672038723594> (burak's thread) - posting copypastas or excerpts from Uncyclopedia.
 <#941708984421847040> (Cringe) - unfunny memes.
-<#909192094055292968> (fizzogs) - posting selfies.
-<#909133916219252756> (Journalist Fetishes) - conversations about imaginary relationships, or "ships".
 <#1094306163283206215> (Baba Yaga's Hut) - conversations about paranormal, esoteric and magical phenomena, as well as pseudoscience, parascience and protoscience. Requires access to <#909046538091778048> (egg-star).
 <#1361967186020929549> - conversations about Romanian politics (mostly in Romanian and English) and other political discussions in Romanian. Requires access to <#909046538091778048> (egg-star).
 ||<#1004866021108220005> - humor too gross for any other channel, but still within the rules. Requires access to <#909046538091778048> (jajo śmierdzi).||
+<#1416853443611590777> - any kind of spam (except for controversial content).
+<#1416853651858657442> (obtrusive ad) - posting advertisements not related to trading such as charitable collections, other servers or sites.
```

Comment:
"Charitable collections" to strasznie dziwna fraza, pierwszy raz taką widzę. Może po prostu:

```suggestion
<#1416853651858657442> (obtrusive ad) - posting advertisements not related to trading such as fundraisers, other servers or sites.
```